scripts/ansible/roles/linux/openenclave/vars/ubuntu/main.yml

```diff
@@ -24,6 +24,7 @@ apt_packages:
   - "autoconf"
   - "doxygen"
   - "graphviz"
+  - "lcov"
```

Comment:
Shall we also install the `lcov-cobertura` pip package?
As a reference this is the PR I had before: https://github.com/openenclave/openenclave/pull/3138/files

Reply:
Are there cases where users would ever need lcov and lcov-cobertura? This Ansible role is used as part of our contributors setup as well and any contributors using those Ansible roles for setup would also install the package.

If the lcov packages are undesirable for contributors I would suggest putting it into the role that is specific to our images: https://github.com/openenclave/openenclave/blob/master/scripts/ansible/roles/linux/jenkins/tasks/packer.yml